citations-api/api/src/java/org/sakaiproject/citation/api/CitationCollectionOrder.java

```diff
@@ -0,0 +1,51 @@
+package org.sakaiproject.citation.api;
+
+/**
+ * Created by nickwilson on 6/23/15.
+ */
+public class CitationCollectionOrder {
+
+	private String collectionId;
+	private int location;
+	private String sectionType;
```

Comment:
But the sectionType can only be a limited number of options so I'd use an enum to make the API easier to use.